playbooks/roles/mongodb/tasks/main.yml

```diff
@@ -91,11 +126,11 @@
         src: mongodb_exporter.service
         dest: /etc/systemd/system/mongodb_exporter.service
 
-    - name: Start MongoDB exporter
+    - name: Start and enable MongoDB exporter
       systemd:
         daemon-reload: yes
         name: mongodb_exporter.service
-        state: started
+        state: restarted
```

Comment:
The generally preferred solution is to use a handler for the restart, and only notify the handler if it's needed, but let's leave it as it is here, since it is good enough for now.

Reply:
Yeah I figured that out, but when I looked in the repo for similar examples I noticed that `restarted` is the common use, so I kept it like this. However, I'll try to change it while addressing the other comment.

Reply:
This one isn't a big deal, so no need to spend much time on it.